.github/workflows/amis-workflow.yml

```diff
@@ -0,0 +1,156 @@
+---
+name: amis-workflow
+
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+    branches:
+      - 'v1.*'
+  schedule:
+    - cron: "0 0 * * 0"
+  # Add pull_request for testing
+  pull_request:
```

Comment:
@quentinDERORY  do we need to remove the pull_request event?

The AWS account to use step (line 39) only supports Schedule or Worflow_dispatch

Reply:
yes remove pull_request event 👍

Reply:
Thanks! I will also remove the lines that I had to force AWS_ACCOUNT = "FB"
that was used for testing too..